Data Structures and Algorithms/Data Structures and AlgorithmsTests/Queue/QueueChallengeTests.swift

```diff
@@ -44,13 +111,13 @@ class QueueChallengeTests: XCTestCase {
         XCTAssertEqual(player3, totalPlayers[3])
         XCTAssertEqual(player4, totalPlayers[4])
         
-        XCTAssertFalse(queue.isEmpty)
+        XCTAssertTrue(queue.isEmpty)
```

Comment:
the queue should not be empty after a round of the game ends. on the contrary it should hold all the player again in the right order